src/Compilers/Core/RebuildTest/CSharpDeterministicKeyBuilderTests.cs

```diff
@@ -80,7 +80,7 @@ static void verifyCount<T>(int expected)
         public void Simple()
         {
             var compilation = CSharpTestBase.CreateCompilation(
-                @"System.Console.WriteLine(""Hello World"");",
+                CSharpTestSource.Parse(@"System.Console.WriteLine(""Hello World"");", checksumAlgorithm: SourceHashAlgorithm.Sha1),
```

Comment:
💭 `CSharpTestBase.CreateCompilation` is already a test helper. Is there any need to change the signature of its callers?

Reply:
Only in this case since this test relies on SHA1 checksum alg, while the test helper uses SHA2 as default.